consul/metadata.csv

```diff
@@ -1,154 +1,143 @@
-consul.memberlist.tcp.accept,gauge,,connection,,This metric counts the number of times a Consul agent has accepted an incoming TCP stream connection.,0,consul,member tcp accept
-consul.memberlist.tcp.connect,gauge,,connection,,This metric counts the number of times a Consul agent has initiated a push/pull sync with an other agent.,0,consul,member tcp connect
-consul.memberlist.tcp.sent,gauge,,byte,,This metric measures the total number of bytes sent by a Consul agent through the TCP protocol,0,consul,member tcp sent
-consul.memberlist.udp.received,gauge,,byte,,This metric measures the total number of bytes sent/received by a Consul agent through the UDP protocol.,0,consul,member udp rec
-consul.memberlist.udp.sent,gauge,,byte,,This metric measures the total number of bytes sent/received by a Consul agent through the UDP protocol.,0,consul,member udp sent
-consul.client.rpc,rate,,request,,"[DogStatsD only] This increments whenever a Consul agent in client mode makes an RPC request to a Consul server. This gives a measure of how much a given agent is loading the Consul servers. Currently, this is only generated by agents in client mode, not Consul servers.",0,consul,client rpc
-consul.client.rpc.failed,gauge,,request,,[DogStatsD only] Increments whenever a Consul agent in client mode makes an RPC request to a Consul server and fails,-1,consul,client rpc fail
-consul.hosts_file.age,gauge,,,,[DogStatsD only] Age of the hosts file,0,consul,hosts_file age
-consul.http.request,gauge,,millisecond,,"[DogStatsD only] This tracks how long it takes to service the given HTTP request for the given verb and path. Using a dogstatsd mapper as described in the README, the paths will be mapped to tags and will not include details like service or key names, for these an underscore will be present as a placeholder (eg. http_method:GET, path:v1.kv._)",-1,consul,http api
-consul.runtime.num_goroutines,gauge,10,,,[DogStatsD only] The number of running goroutines,0,consul,num goroutines
-consul.runtime.alloc_bytes,gauge,10,byte,,[DogStatsD only] Current bytes allocated by the Consul process,0,consul,bytes alloc
-consul.runtime.heap_objects,gauge,10,object,,[DogStatsD only] The number of objects allocated on the heap,0,consul,heap objs
-consul.runtime.sys_bytes,gauge,10,byte,,[DogStatsD only] Total size of the virtual address space reserved by the Go runtime,0,consul,run sys bytes
-consul.runtime.malloc_count,gauge,10,object,,[DogStatsD only] Cumulative count of heap objects allocated,0,consul,run malloc cnt
-consul.runtime.free_count,gauge,10,object,,[DogStatsD only] Cumulative count of heap objects freed,0,consul,run free cnt
-consul.runtime.total_gc_pause_ns,gauge,10,nanosecond,,[DogStatsD only] Cumulative nanoseconds in GC stop-the-world pauses since Consul started,0,consul,run tot gc pause
-consul.runtime.total_gc_runs,gauge,10,,,[DogStatsD only] The number of completed GC cycles,0,consul,run total gc runs
```

Comment:
These metrics are not documented in the readme for dogstatsd, and we don't submit them with prometheus because of their current format having an idea in the middle of the name
`consul_810a39248433_runtime_alloc_bytes 2.138024e+07`